app/views/main_without_nav.scala.html

```diff
@@ -0,0 +1,224 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.Play
+@import play.api.Play.current
+@import uk.gov.hmrc.http.SessionKeys
+@import uk.gov.hmrc.play.views.helpers.AttorneyRegime.standAlone
+@import uk.gov.hmrc.play.views.html.layouts
+@import uk.gov.hmrc.tai.util.constants.TaiConstants
+@import uk.gov.hmrc.tai.config.{ApplicationConfig, FeatureTogglesConfig}
+@import uk.gov.hmrc.play.language.LanguageUtils
+@import uk.gov.hmrc.tai.viewModels.GoogleAnalyticsSettings
+
+@import uk.gov.hmrc.play.views.helpers.AttorneyRegime
+
+@(title: String,
```

Comment:
This whole file seems unnecessary, can we just use the account menu to be disabled in mustacheMap?

Reply:
@Muhammad-Ali-Ahmad Can you please have a look at this?

Reply:
@Christian-QA Ste is going to look in to this and suggest how to improve it